burn-book/src/import/onnx-model.md

```diff
@@ -0,0 +1,135 @@
+# Import ONNX Model
+
+## Why Importing Models is Necessary
+
+In the realm of deep learning, it's common to switch between different frameworks depending on your
+project's specific needs. Maybe you've painstakingly fine-tuned a model in TensorFlow or PyTorch and
+now you want to reap the benefits of Burn's unique features for deployment or further testing. This
+is precisely the scenario where importing models into Burn can be a game-changer.
+
+## Traditional Methods: The Drawbacks
+
+If you've been working with other deep learning frameworks like PyTorch, it's likely that you've
+exported model weights before. PyTorch, for instance, lets you save model weights using its
+`torch.save()` function. Yet, to port this model to another framework, you face the arduous task of
+manually recreating the architecture in the destination framework before loading in the weights. Not
+only is this method tedious, but it's also error-prone and hinders smooth interoperability between
+frameworks.
```

Comment:
Sometimes it's necessary and that's fine. When using framework specific features, it's just not a possibility to serialize into a standard; the standard is always behind new innovations. So I would just reframe a bit and enphasis on _standard_ models.